test/libyul/yulOptimizerTests/fullSuite/aztec.yul

```diff
@@ -245,17 +245,16 @@
 //         mstore(0x00, 404)
 //         revert(0x00, 0x20)
 //     }
-//     let validateJo_kn_287 := calldataload(add(calldatasize(), 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff40))
-//     let validateJo_kn := validateJo_kn_287
+//     let validateJo_kn := calldataload(add(calldatasize(), 0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff40))
 //     let validateJo__24 := 0x2a0
 //     mstore(validateJo__24, caller())
-//     mstore(0x2c0, validateJo_kn_287)
+//     mstore(0x2c0, validateJo_kn)
 //     mstore(0x2e0, validateJo_m)
-//     validateJo_kn := mulmod(sub(validateJo_gen_order, validateJo_kn_287), validateJo_challenge, validateJo_gen_order)
+//     validateJo_kn := mulmod(sub(validateJo_gen_order, validateJo_kn), validateJo_challenge, validateJo_gen_order)
 //     hashCommitments(validateJo_notes, validateJo_n)
 //     let validateJo_b := add(0x300, mul(validateJo_n, validateJo__6))
-//     let validateJo_i_290 := 0
-//     let validateJo_i := validateJo_i_290
+//     let validateJo_i := 0
+//     let validateJo_i_1306 := validateJo_i
```

Comment:
Did you check why this is still here?

Reply:
Ah, `validateJo_i` is the loop variable, and `validateJo_i_1306` is just a zero constant.

Reply:
To make code better readable, we could rename variables that are static constants to meaningful names.